examples/tensorflow2/tensorflow2.ipynb

```diff
@@ -0,0 +1,644 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Parse model response to find the highest prediction probability."
```

Comment:
Since it's not only for the next cell. Could we change `Parse model response to find the highest prediction probability.` to like `In the following cells, we will parse the model response to find the highest prediction probability.`